example/app/app.component.html

```diff
@@ -3,20 +3,27 @@ <h1>&lt;angular-custom-modal&gt;</h1>
 
   <div id="actions">
     <button class="big" (click)="componentInsideModal.open()">Component inside modal</button>
-    <!-- Alternatively we can define the open call on the component 
+    <!-- Alternatively we can define the open call on the component
       <button class="big" (click)="openFromComponent()"></button> -->
     <button class="big" (click)="htmlInsideModal.open()">Raw HTML inside modal</button>
+    <button class="big" (click)="htmlInsideModalWhitNestedModal.open()">Modal with nested modal</button>
```

Comment:
Typo -> "With"

Reply:
just realize that, my mistake, I speak Spanish. Easy TODO

Reply:
corrected